core/src/main/scala/org/apache/spark/api/python/PythonWorkerFactory.scala

```diff
@@ -17,7 +17,7 @@
 
 package org.apache.spark.api.python
 
-import java.io.{DataInputStream, DataOutputStream, InputStream, OutputStreamWriter}
+import java.io._
```

Comment:
BTW, don't forget to get rid of this import change too. This import looks needed due to `IOException`.

Reply:
To save on jenkins resources :), I want to verify that the latest version of the message is OK:

<pre>
org.apache.spark.SparkException:
Bad data in pyspark.daemon's standard output. Invalid port number:
  1315905645 (0x4e6f206d)
Python command to execute the daemon was:
  python -m pyspark.daemon
Check that you don't have any unexpected modules or libraries in
your PYTHONPATH:
  /.../github/spark_fork/python/lib/pyspark.zip:/.../github/spark_fork/python/lib/py4j-0.10.6-src.zip:/.../github/spark_fork/assembly/target/scala-2.11/jars/spark-core_2.11-2.4.0-SNAPSHOT.jar:/.../github/spark_fork/python/lib/py4j-0.10.6-src.zip:/.../github/spark_fork/python/:
Also, check if you have a sitecustomize.py module in your python path,
or in your python installation, that is printing to standard output

</pre>

Reply:
👍